LICENSE

```diff
@@ -706,3 +706,247 @@
     FOR ANY DAMAGES OR OTHER LIABILITY, WHETHER IN CONTRACT, TORT OR OTHERWISE,
     ARISING FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
     DEALINGS IN THE SOFTWARE.
+
+
+
+    =======================================================================================
+    Appendix
+    =======================================================================================
+
+                                 Apache License
```

Comment:
How is this different than the Apache 2.0 license at the beginning of the file?

Reply:
They are there for different reasons. The one at the top is the license for the MXNet project. The one at the bottom is part of the list of fully written out licenses that are used by the third party projects. I thought it would be a bit odd to have it say to find the licenses in the Appendix and then the appendix says look at the top of the file.